src/views/datatiles/data-tiles.html

```diff
@@ -0,0 +1,14 @@
+<div class="tiles-view-pf">
+  <div class="tile" ng-repeat="item in items"
+       style="width:{{config.width}}px; height:{{config.height}}px;"
```

Comment:
It seems cleaner to me and doesn't result in inline styles in case someone wanted to override the style.  I'm not sure if setting of height and width of the tile is best handled this way (as opposed to just setting it with CSS) but I'm not really clear on the requirements for this component.